odoo/addons/base/models/ir_model.py

```diff
@@ -28,6 +28,8 @@
     'time': time,
 }
 
+# pylint: disable=raise-unlink-override
```

Comment:
the occurrences are low-level, legit cases and are guarded by a check such as:

```python
if not self.env.context(MODULE_UNINSTALL_FLAG):
    raise UserError(...)
```

Reply:
This file is 2250 lines.  Simply stating "don't worry about this file" is way too dangerous, IMHO.
Disabling the check on a specific method is acceptable, but not on a whole file!

Reply:
indeed, I ended up extracting the raises to their own ondelete methods